### ISSUE TITLE
Question: May I know whether this project is still active

### ISSUE THREAD

**Issue by lblim** [solettaproject/soletta-visual-editor#11]
May I know whether this project is still active? Will it be included into meta-soletta as part of the release?

**Comment by barbieri** [comment]
no, it's not active and no manpower to work on this part of the project.

We know it's an important bit, but to be really usable demands lots of manpower to be done, something we don't have :-/

Shall you want to develop or sponsor its development, let me know.

**Comment by lblim** [comment]
This is a good feature. On high level estimate, how much manpower will it take to make it usable?

**Comment by barbieri** [comment]
@lblim it depends on the features you want implemented and the timeline for each feature. It can range from a very basic visual editor without lots of graphical effects and no special handling for meta types (used in `DECLARE=` statements) to a complete solution with that, real time inspector and so on.

A simple editor could be achieved in 8-12 months of 3-5 engineers, 1 designer and 1 QA. This is a rough estimate, we could refine it and get a more realistic estimate based on features.
